tests/manage/z_cluster/test_rook_ceph_osd_flapping.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import pytest
+
+
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.cluster import ceph_health_check
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+    ignore_leftovers,
+)
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@tier2
+@ignore_leftovers
+@bugzilla("2223959")
+@skipif_external_mode
+@skipif_ocs_version("<4.14")
+@pytest.mark.polarion_id("OCS-XXXX")
+class TestRookCephOsdFlapping(ManageTest):
+    """
+    Test Rook Ceph OSD Flapping
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        def finalizer():
+            ceph_health_check()
+
+        request.addfinalizer(finalizer)
+
+    def test_rook_ceph_osd_flapping(self):
+        """
+        Test Process:
+
+        1.Get osd id
+        2.Mark an osd down manually
+        3.Verify osd is down
+        4.Verify osd log contain "osd_max_markdown_count 5 in last" string
+        5.Scale up osd
+        6.Verify ceph status is OK
+        """
+        log.info("Get One OSD ID")
+        osd_pod_objs = pod.get_osd_pods()
+        self.osd_pod_id = pod.get_osd_pod_id(osd_pod_objs[0])
+        ct_pod = pod.get_ceph_tools_pod()
+        log.info(f"Mark an osd {self.osd_pod_id} down manually")
+        ct_pod.exec_ceph_cmd(f"ceph osd down osd.{self.osd_pod_id}")
+
```

Comment:
the verification step in the bz says "1. Mark an osd down manually in a quick succession 5 times:". Where is this taken care of?...

Reply:
[done](https://github.com/red-hat-storage/ocs-ci/pull/9082/files#diff-126caffcf12de0bc2205862b21557de21faa681bf2699637fa0ff2a7d85c273cR60)